blips/blip-0001.mediawiki

```diff
@@ -0,0 +1,236 @@
+<pre>
+bLIP: 1
+Title: bLIP process
+Status: Active
+Type: Process
+Author: Ryan Gentry <ryan@lightning.engineering>, *ideally plus one person from each client team*
+Created: 2021-05-21
+License: BSD-2-Clause, OPL
+</pre>
+
+==Abstract==
+
+bLIP stands for Bitcoin Lightning Improvement Proposal. A bLIP is a design document providing information to the Lightning community, or describing a new feature for the Lightning Network or its processes or environment. The bLIP should provide a concise technical specification of the feature and a rationale for the feature. The bLIP author is responsible for building consensus within the community and documenting dissenting opinions.
+
+==Copyright==
+
+This bLIP is dual-licensed under the Open Publication License and BSD 2-clause license.
+
+==Rationale==
+
+As the Lightning community has grown, new features, standards, and protocols have been developed outside of the BOLT specification process: particularly at the application, forwarding, and routing level that isn’t described within the core BOLT documents. This is great! But in the spirit of interoperability, documenting features, standards, and protocols in a single location with a standard format will make it easy on future adopters. 
+
+bLIPs will serve as the primary mechanism for proposing new features, for collecting community input on an issue, and for documenting the design decisions that have gone into the Lightning Network. Hopefully, they will provide an avenue for developers to quickly get feedback on their ideas outside of the main BOLT process. Because the bLIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+For Lightning developers, bLIPs are a convenient way to track the progress of their implementation. Ideally, each implementation editor would list the bLIPs they have implemented. This will give end users a convenient way to know the current status of a given implementation or library.
+
+==bLIP Types==
```

Comment:
I don't know if we need to specify these types now, as we don't know yet how exactly blips will be used.
These types will likely be too narrow or too broad.
I'd rather remove all this and amend `blip-0001` later when we have accepted a few blips and have a better idea of how to create the right categories to sort them.

Overall I'm in favor of making this file as simple as possible at first (and as concise as possible), and enrich it as we go, otherwise there's a chance we'll never even agree on blip-0001 which slows down the whole process.